.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+compute_engine_instance:
+  image_project: freebsd-org-cloud-dev
+  image: family/freebsd-13-1
+  platform: freebsd
+  disk: 100 # Gb
+
+build_task:
+  name: build
+  timeout_in: 120m
+  only_if: $CIRRUS_TAG != ''
+  env:
+    AWS_ACCESS_KEY_ID: ENCRYPTED[25d5e1d29570da70ceeb75e842b98be63d110824997b3c6b7bc0735b8acc686fbe0e9995cfe4d22a61d707a2e705d48b]
```

Comment:
What are these secrets and who has access to them?

Reply:
Those are secrets for S3 access to my linode bucket. I have access to them.